client/src/components/nav/Nav.css

```diff
@@ -13,16 +13,18 @@ nav {
   color: #e1e1e1;
   text-decoration: none;
   /* margin-top: 50px; */
-  padding: 20px;
+  padding: 10px;
+
   
 }
 
 .auth a {
   color: #e1e1e1;
   text-decoration: none;
-  margin-right: 100px;
-  padding: 20px;
-  
-
+  margin-right: 20px;
+  padding: 10px;
+}
 
-}
\ No newline at end of file
+/* .lhlogo {
```

Comment:
Do you still need this or can we delete it?

Reply:
yep, i can delete